Source/Scene/ModelExperimental/TextureManager.js

```diff
@@ -0,0 +1,170 @@
+import defined from "../../Core/defined.js";
```

Comment:
Out of curiosity have you tried loading a KTX2 texture into a custom shader? See some examples in https://sandcastle.cesium.com/?src=Materials.html

Reply:
This is not currently supported to keep the scope of this PR focused on the essentials.

Though how important is supporting KTX2 here? can it wait until #9741?